yarn.lock

```diff
@@ -2472,6 +2472,46 @@
     remark-gfm "^1.0.0"
     zen-observable "^0.8.15"
 
+"@backstage/core@^0.4.3":
```

Comment:
a couple of unrelated things in here - please try to

- pull latest master
- rebase your branch on top of that
- check out the unaltered master yarn.lock locally
- then run `yarn install` again